### PR TITLE
Use shared jaeger-compose in xdock

### PR DESCRIPTION
**Pull request by pavolloffay**

### CODE REVIEW

File: jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java [hunk 1]
```diff
@@ -49,6 +49,10 @@
 import org.glassfish.jersey.server.ResourceConfig;
 
 public class JerseyServer {
+  private static final String SAMPLING_HOST_PORT = "SAMPLING_HOST_PORT";
+  private static final String AGENT_HOST = "AGENT_HOST";
+  private static final String COLLECTOR_HOST_PORT = "AGENT_HOST";
```
Comment (black-adder) [inline jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java]: is this a typo or am I missing something?

Comment (pavolloffay) [reply jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java]: it's a typo, fixed. 

Currently it's not used, I wanted to make all this configurable.

File: jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java [hunk 2]
```diff
@@ -143,12 +149,20 @@ public static void main(String[] args) throws Exception {
...
-  private static Sender senderFromEnv(String jaegerHost) {
+  private static String getEvn(String envName, String defaultValue) {
+    String env = System.getenv(envName);
+    if (env == null) {
+      return defaultValue;
+    }
+    return env;
+  }
+
+  private static Sender senderFromEnv(String collectorHostPort, String agentHostPort) {
```
Comment (black-adder) [inline jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java]: I'm being pedantic here but should s/agentHostPort/agentHost

Comment (pavolloffay) [reply jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java]: a little bit ;) I will update

**Comment by pavolloffay** [comment]
@black-adder I will merge to be able to clean other xdocks. If there is anything more I will fix in the following PR.

**Pull request merged**